changelog/2674.bugfix.rst

```diff
@@ -0,0 +1 @@
+Fixed reading pieces of data larger than 16 KiB on macOS.
```

Comment:
Can we be specific and say this issue affects SecureTransport on macOS.

Reply:
Yes, I updated the entry. Also, I removed the exact amount because I remembered that I saw SecureTransport reading 8 and 32 KiB at a time from real resources while I was debugging the problem.